src/components/LabeledRadioButton/examples/states.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import LabeledRadioButton from '../LabeledRadioButton';
+
+const style = {
+	marginRight: '5px'
+};
+
+export default React.createClass({
+	render() {
+		return (
+			<section style={{ display: 'flex' }}>
+				<LabeledRadioButton style={style} />
+				<LabeledRadioButton Label='String passed in as the label' style={style} />
+				<LabeledRadioButton Label={<p>HTML element passed in as the label</p>} style={style} />
```

Comment:
This example should be replaced with one that passes `Label` as a child element with children rather than passing an element thru props.